App/components/Input/input.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useCallback, useRef } from 'react'
+import { TextInput } from 'react-native'
+import {
+  Container,
+  PlaceholderLabel,
+  LabeledInput,
+  RubleIcon,
+} from './input.styled'
+import Ruble from '../../../assets/images/ruble.svg'
+
+interface InputProps {
+  placeholder: string
+}
+
+interface InputReference extends TextInput {
```

Comment:
Check if we can use type extension via ```TextInputProps``` instead of interface

Reply:
done